main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
```

Comment:
Unfortunately, this doesn't actually comment out code. Replace // with # in order to do so!